tensorboard/components/tf_categorization_utils/tf-category-pane.html

```diff
@@ -0,0 +1,212 @@
+<!--
+@license
+Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../iron-collapse/iron-collapse.html">
+<link rel="import" href="../polymer/polymer.html">
+
+<dom-module id="tf-category-pane">
+  <template>
+    <template is="dom-if" if="[[_rendered]]">
+      <button class="heading"
+              on-tap="_togglePane"
+              open-button$="[[_opened]]">
+        <span class="name">
+          <template is="dom-if" if="[[_isSearchResults]]"><!--
+            --><span class="light">Tags matching /</span><!--
+          --></template><!--
+          -->[[category.name]]<!--
+          --><template is="dom-if" if="[[_isSearchResults]]"><!--
+            --><span class="light">/</span><!--
```

Comment:
These are `light` to distinguish the static GUI text from the actual tag name, but I'm happy to consider alternative approaches.